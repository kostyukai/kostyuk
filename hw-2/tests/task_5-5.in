0 1 2 3 10
1.2 2.84 2.16 7.8 9.51
