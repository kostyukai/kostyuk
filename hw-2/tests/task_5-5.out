44
20.00
